docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
+
+The Entity Analytics dashboard provides a centralized view of critical entity metrics, which include hosts, users, and anomalies. Use this information to determine which emerging threats pose a higher risk to your environment so you can respond quickly. 
```

Comment:
The Entity Analytics dashboard provides a centralized view into emerging insider threats - including host and user risk and notable anomalies from within your network. Use these analytics to triage emerging threats, investigate and respond.